metrics_utility/management/commands/build_report.py

```diff
@@ -109,24 +110,26 @@ def handle(self, *args, **options):
         extra_params['month_until'] = next_month
         extra_params['deduplicator'] = os.getenv('METRICS_UTILITY_DEDUPLICATOR') or None
 
-        extractor = ExtractorFactory(ship_target, extra_params).create()
-
         # Determine destination path for generated report and skip processing if it exists
+        report_type = extra_params['report_type']
+        ship_path = extra_params['ship_path']
         if opt_since is not None:
-            now = datetime.datetime.now().replace(second=0, microsecond=0, tzinfo=timezone.utc)
-            extra_params['since_date'] = opt_since.date()
-            extra_params['until_date'] = opt_until.date() if opt_until else now.date()
+            since_date = opt_since.date()
+            until_date = opt_until.date() if opt_until else datetime.date.today()
```

Comment:
This is practicaly removing datetime granularity and setting date granularity.

Are you sure we will not need it in service? I can imagine we will be building reports for hours.